numpy/fft/_pocketfft.py

```diff
@@ -80,12 +80,12 @@ def _raw_fft(a, n, axis, is_real, is_forward, inv_norm):
 
 
 def _unitary(norm):
-    if norm is None:
-        return False
-    if norm=="ortho":
+    if norm == "ortho":
         return True
-    raise ValueError("Invalid norm value %s, should be None or \"ortho\"."
-                     % norm)
+    if norm is None or norm == "inverse":
+        return False
+    raise ValueError("Invalid norm value %s; should be None, \"ortho\" or\
+                    \"inverse\"." % norm)
```

Comment:
Since we touch it, maybe just use f-string and maybe also r-string.

Reply:
changed to f-string instead